src/test/java/org/folio/rest/impl/MockServer.java

```diff
@@ -362,25 +363,30 @@ private void handleGetVoucherLineById(RoutingContext ctx) {
   }
   
   private void handleGetInvoiceLineById(RoutingContext ctx) {
-    logger.info("handleGetInvoiceLinesById got: GET " + ctx.request().path());
-    String id = ctx.request().getParam(ID);
+    logger.info("handleGetInvoiceLinesById got: GET " + ctx.request()
+      .path());
+    String id = ctx.request()
+      .getParam(ID);
     logger.info("id: " + id);
     if (ID_FOR_INTERNAL_SERVER_ERROR.equals(id)) {
       serverResponse(ctx, 500, APPLICATION_JSON, Response.Status.INTERNAL_SERVER_ERROR.getReasonPhrase());
     } else {
       try {
         String filePath = String.format(MOCK_DATA_PATH_PATTERN, INVOICE_LINES_MOCK_DATA_PATH, id);
-  
         JsonObject invoiceLine = new JsonObject(getMockData(filePath));
-  
         // validate content against schema
-       InvoiceLine invoiceSchema = invoiceLine.mapTo(InvoiceLine.class);
+        InvoiceLine invoiceSchema = invoiceLine.mapTo(InvoiceLine.class);
         invoiceSchema.setId(id);
+        if (id.equals(INVOICE_LINE_VERIFY_TOTAL)) {
```

Comment:
In `getInvoicingInvoiceLinesByIdTest` test you check that existingTotal=2.00, and here you set it to 2.40. That makes the tests not fair.